src/Layer/GeometryLayer.js

```diff
@@ -171,7 +171,7 @@ class GeometryLayer extends Layer {
      * All layer's 3D objects are removed from the scene and disposed from the video device.
      * @param {boolean} [clearCache=false] Whether to clear the layer cache or not
      */
-    delete(clearCache) {
+    dispose(clearCache) {
         if (clearCache) {
             this.cache.clear();
```

Comment:
All Layers implementing this method start by:
```
if (clearCache) {
            this.cache.clear();
}
``` 

Could you factorize it in `Layer.dispose` please ? 

Also, you need to replace `this.cache.clear()`  by `this.cache.dispose()`.